fms2_io/readme.md

```diff
@@ -88,7 +90,7 @@ endif
   - The "x" and "y" argument indicate that that dimension is domain decomposed in x/y. The only acceptable values are "x" and "y".
   - The `position=center` indicates the position of the axis (this is the default). The other acceptable values are `position=east` for "x" and`position=north` for "y", in this cases the axis will have an extra point.
```

Comment:
May want to use the term staggered data which may be on the volume face or corner instead of at the centroid.

Reply:
So like this?

The other acceptable values are `position=east` for "x" and`position=north` for "y", in this cases the data is staggered, which may be on the volume face or corner instead of at the centroid.

Reply:
@uramirez8707 that sounds good.